src/services/types.ts

```diff
@@ -32,7 +32,7 @@ namespace ts {
         getEscapedName(): __String;
         getName(): string;
         getDeclarations(): Declaration[] | undefined;
-        getDocumentationComment(): SymbolDisplayPart[];
+        getDocumentationComment(typeChecker?: TypeChecker): SymbolDisplayPart[];

```

Comment:
Based on https://github.com/Microsoft/TypeScript/pull/19455#discussion_r147214986 maybe it would be better to just take a compile-time breaking change and make this `typeChecker: TypeChecker | undefined`. @mhegazy